packages/headless/src/controllers/static-filter/headless-static-filter.ts

```diff
@@ -65,6 +70,14 @@ export interface StaticFilter extends Controller {
    * */
   deselectAll(): void;
 
+  /**
+   * Checks whether the specified static filter value is selected.
+   *
+   * @param value - The static filter value to check.
+   * @returns Whether the specified static filter value is selected.
+   */
+  isValueSelected(value: StaticFilterValue): boolean;
```

Comment:
I don't know if this is necessary, especially if we take the object as a parameter.

Reply:
Yeah, that seems very superfluous as it currently stands.

Reply:
I'm going to keep the method ... If you feel it should be removed, I can do a pass through the static filter and facet controllers and deprecate it in a separate PR.